docker/scripts/deprovision.sh

```diff
@@ -163,8 +163,15 @@ if [[ $preserve_data == false ]]; then
 				done
 			fi
 
-			# flbas 0 uses 512 byte sector sizes
-			sectors=$((max_bytes / 512))
+			nvmemodel=$(nvme id-ctrl "$drive" -o json | jq -r '.mn' | sed -e 's/[[:space:]]*$//')
+			if [[ $nvmemodel == 'INTEL SSDPE2KX040T8' ]]; then
+				# Set specific block size depending on physical BD
+				sectors=$((max_bytes / 4096))
+			else
+				# default flbas 0 uses 512 byte sector sizes
+				sectors=$((max_bytes / 512))
```

Comment:
nvme tool doesn't have a block size output we can use instead of hard coding cards?

Reply:
My thoughts exactly. The nvme tool does have it but its something that gets removed at deprov. I reluctant to hard code the disk model (like we still do for HW raid controllers) but this is what we have right now that works.

Reply:
how do we get it back? Do we need to create a namespace first?